tracing-subscriber/src/filter/env/mod.rs

```diff
@@ -189,22 +189,32 @@ impl EnvFilter {
     /// directives, either added using this method or provided when the filter
     /// is constructed.
     ///
-    /// Filters may be created from may be [`LevelFilter`]s, which will
+    /// Filters may be created from [`LevelFilter`] or [`Level`], which will
     /// enable all traces at or below a certain verbosity level, or
     /// parsed from a string specifying a directive.
     ///
     /// If a filter directive is inserted that matches exactly the same spans
     /// and events as a previous filter, but sets a different level for those
     /// spans and events, the previous directive is overwritten.
     ///
-    /// [`LevelFilter`]: struct.LevelFilter.html
+    /// [`LevelFilter`]: ../filter/struct.LevelFilter.html
+    /// [`Level`]: ../../tracing_core/struct.Level.html
```

Comment:
does this link work on docs.rs? i think relative URLs like this will work locally & on netlify, since we are building all the other docs in the same target dir, but i don't think it will work on docs.rs. typically, i use the full docs.rs URL when linking across crates.

Reply:
When I built with `cargo doc --open` it did. I actually copied it from the top of the same file, there's a bunch that use that path. If it doesn't work on docsrs we may want to change the links at the top too.

Reply:
Yeah so I just tried https://docs.rs/tracing-subscriber/0.2.11/tracing_subscriber/filter/struct.EnvFilter.html and none of the links at the top work on docsrs.

So, what should I put here? literally `https://docs.rs/tracing-core/0.1.14/tracing_core/metadata/struct.Level.html` ?

Reply:
> So, what should I put here? literally `https://docs.rs/tracing-core/0.1.14/tracing_core/metadata/struct.Level.html` ?

Yup, that's right.